### ISSUE TITLE
Landmarks nav element not seen

### ISSUE THREAD

**Issue by jstallent** [IDPF/epubcheck#457]
I got this HINT message when running a test file through alpha 11:
<message>ACC-008, HINT, [Navigation Document has no 'landmarks nav' element.], BookOfTexas_ePub3_v3.epub</message>

However, that EPUB has a landmarks nav element, located at the end of the nav doc after the pagelist and some other nav lists:

```
<nav epub:type="landmarks" hidden="">
<h1 class="fm">Guide</h1>
<ol>
    <li><a epub:type="cover" href="1-cover.xhtml#cover">Cover</a></li>
    <li><a epub:type="toc" href="6-TOC.xhtml#toc">Table of Contents</a></li>
    <li><a epub:type="bodymatter" href="7-start.xhtml#start">Start Reading</a></li>
</ol>
</nav>
```

**Comment by jenstroeger** [comment]
I'm seeing the same problem here: with or without a [`"landmark"`](https://ebookflightdeck.com/handbook/landmarks) `nav` element in my toc.xhtml file, I get that same usage message: `USAGE(ACC-008): mn-9.epub/mn-9.epub(-1,-1): Navigation Document has no 'landmarks nav' element.`

What's the status on this issue?

**Comment by tofi86** [comment]
This hadn't been working because of a typo in the EpubChecl source code: `landmark` instead of `landmarks` was checked (5994ac3).

There has also been another issue with checking landmarks nav elements, so I fixed this and alsoanother issue (#734) in PullRequest #735 which will be included in the next release...
